src/components/app/MenuButtons.js

```diff
@@ -173,7 +173,12 @@ function _formatVersionNumber(version?: string): string | null {
 function _formatLabel(meta: ProfileMeta): string | null {
   const product = meta.product || '';
   const version = _formatVersionNumber(meta.misc) || '';
-  const os = meta.oscpu || '';
+  const os = (() => {
+    if (meta.platform !== undefined && meta.platform.match(/android/i)) {
+      return meta.platform;
+    }
+    return meta.oscpu || '';
+  })();
```

Comment:
It works well but arrow function looks a bit overkill to me. Yeah, it's good to use `const` variables in general but there is no shame using `let` as well :) We generally go with simpler approaches. If you look at the codebase you can see some examples of this type of conditional assignments:
```js
let os;
if (meta.platform !== undefined && meta.platform.match(/android/i)) {
  os = meta.platform;
} else {
  os = meta.oscpu || '';
}
```
I think that will make the code more readable and straightforward. (I know I'm a bit nitpicky, sorry :))
Also it would use a comment on top of `if` that explains we are doing this because we want to display android version instead of linux since it's more important for android developers.

Reply:
Actually I was worried about using let, so thank you for clarifying that for me :)